airflow/serialization/schema.json

```diff
@@ -103,7 +109,8 @@
           { "type": "null" },
           { "$ref": "#/definitions/task_group" }
         ]},
-        "edge_info": { "$ref": "#/definitions/edge_info" }
+        "edge_info": { "$ref": "#/definitions/edge_info" },
+        "dependencies": { "$ref": "#/definitions/dependencies" }
```

Comment:
We should probably call this `dag_dependencies` WDYT?

Reply:
can change